show_client/interface_cli.go

```diff
@@ -382,3 +382,44 @@ func getInterfaceFecStatus(options sdc.OptionMap) ([]byte, error) {
 
 	return json.Marshal(portFecStatus)
 }
+
+func getInterfaceAlias(options sdc.OptionMap) ([]byte, error) {
+    intf, _ := options["interface"].String()
+
+    // Read CONFIG_DB.PORT
+    queries := [][]string{{"CONFIG_DB", "PORT"}}
+    portEntries, err := GetMapFromQueries(queries)
+    if err != nil {
+        log.Errorf("Failed to get ports from CONFIG_DB: %v", err)
+        return nil, err
+    }
+
```

Comment:
Can we sort port entries as done in show interface alias. I believe this is already done by others.

```
ports = natsortInterfaces(ports)
```